docs/annotations.md

```diff
@@ -0,0 +1,33 @@
+# Annotations
+
+The following annotations are used by the operator to control the injection and define configuration of the flagd sidecar.
+
+### `openfeature.dev/enabled`
+When a value of `"true"` is provided, the operator will inject a flagd sidecar into the annotated pods.  
+Example: 
+```
+    metadata:
+    annotations:
+        openfeature.dev/enabled: "true"
+```
+
+### `openfeature.dev/featureflagconfiguration`
+This annotatation defines the FeatureFlagconfiguration CRD that is to be used by the flagd sidecar, only the name of the CRD should be passed, it is expected that the CRD is deployed to the same `namespace` as the pod.  
```

Comment:
```suggestion
This annotation defines the FeatureFlagConfiguration CRD that is to be used by the flagd sidecar, only the name of the CRD should be passed, it is expected that the CRD is deployed to the same `namespace` as the pod.  
```

Reply:
**Aside**
It actually _specifies the name_ of a FeatureFlagConfiguration custom resource.

The CRD is what makes the FeatureFlagConfiguration API appear in your cluster. A named FeatureFlagConfiguration is a custom resource, not a custom resource definition. CRDs are not namespaced; the APIs that they define might be.